tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/AddressSteps.cs

```diff
@@ -52,11 +52,11 @@ public async Task GivenAddressesExist(Table table)
             _context[ScenarioContextKeys.AddressMapDictionary] = addressDictionary;
         }
 
-        [Then(@"the Address section (.*) is returned")]
-        public async Task ThenTheAddressSectionIsReturned(string section, Table table)
+        [Then(@"the Address section is returned")]

```

Comment:
Is it clearer if the word section is removed? As it could now be classed as redundant.

Reply:
👍